pkg/tempo/config.go

```diff
@@ -29,6 +32,21 @@ import (
 const (
 	spanMetricsPipelineName    = "metrics/spanmetrics"
 	defaultSpanMetricsExporter = "prometheus"
+
+	// defaultDecisionWait is the default time to wait for a trace before making a sampling decision
+	defaultDecisionWait = time.Second * 5
+
+	// defaultLoadBalancingPort is the default port the agent uses for internal load balancing
+	defaultLoadBalancingPort = "9999"
```

Comment:
maybe we should choose otlp default + 1? would give it a bit more meaning.

Reply:
Yea, 9999 was just a placeholder. Changed in 6ca6e3efde5d4d9b5f5847d0af0362be5d5adf08